### PR TITLE
Tahoe Quickstart instructions

### PR DESCRIPTION
**Pull request by OmarIthawi**

### CODE REVIEW

**Comment by johnbaldwin** [comment]
@OmarIthawi Thanks! I'll add comments as I stand up hawthorn devstack

**Comment by OmarIthawi** [comment]
Please do @johnbaldwin! This Tahoe btw!

**Comment by OmarIthawi** [comment]
[(6) Redirect issue and basic tests for the Hawthorn - Omar's Hawthorn upgrade buffer 52.2](https://trello.com/c/EvpqFlWS/3552-6-redirect-issue-and-basic-tests-for-the-hawthorn-omars-hawthorn-upgrade-buffer-522)

**Comment by johnbaldwin** [comment]
I'll create a card for next sprint to walk through the setup process again to make sure the Mac specific steps are accurate.

I think there are more steps needed to get devstack installed than outlined in this PR. When I try installing the devstack again, I can better capture the steps now that I've gotten a bit more familiar with it

**Comment by OmarIthawi** [comment]
@johnbaldwin After https://github.com/appsembler/amc/pull/159 you should have no libssl and compilation issues anymore.

Could you please review this pull request again, it should be ready to get merged.

File: README.rst [hunk 1]
```diff
@@ -3,9 +3,101 @@ Open edX Devstack |Build Status|
...
 ===================
-To run a Tahoe devstack, follow the same instructions in the "Prerequisites" section,
-but use ``$ make tahoe.up`` instead of ``$ make dev.up`` to use the Tahoe specific
-features and environment settings.
+To run a Tahoe devstack follow the steps below:
+
+This project requires **Docker 17.06+ CE**.
+
+.. code::
+
```
Comment (johnbaldwin) [inline README.rst]: Are we cloning this repo? Which should be our default branch?

File: README.rst [hunk 2]
```diff
@@ -3,9 +3,101 @@ Open edX Devstack |Build Status|
...
+    $ make dev.clone
+    $ make dev.provision
```
Comment (johnbaldwin) [inline README.rst]: I cloned the devstack appsembler fork. Did not change branch (so branch is `hawthorn`)
After cloning the appsembler devstack fork and cd'ing to that directory, I ran just the commands specified above

`make dev.clone` This seems to have worked fine

`make dev.provision` this seems to have worked well until here:

```
---> pavelib.assets.webpack
python manage.py lms --settings=devstack_docker print_setting STATIC_ROOT 2>/dev/null
python manage.py cms --settings=devstack_docker print_setting STATIC_ROOT 2>/dev/null
python manage.py lms --settings=devstack_docker print_setting WEBPACK_CONFIG_PATH 2>/dev/null
NODE_ENV=development STATIC_ROOT_LMS=DISABLING CACHES...
/edx/var/edxapp/staticfiles STATIC_ROOT_CMS=DISABLING CACHES...
/edx/var/edxapp/staticfiles/studio $(npm bin)/webpack --config=DISABLING CACHES...
webpack.dev.config.js
/bin/sh: 1: CACHES...: not found
/bin/sh: 2: /edx/var/edxapp/staticfiles: Permission denied
/bin/sh: 3: /edx/var/edxapp/staticfiles/studio: Permission denied
/bin/sh: 4: webpack.dev.config.js: not found

...

Build failed running pavelib.assets.update_assets: Subprocess return code: 127
make: *** [dev.provision.run] Error 1
```

Comment (johnbaldwin) [reply README.rst]: I started over with a clean install and get the same error

Comment (thraxil) [reply README.rst]: @johnbaldwin confirming that I get the same error on a clean `make dev.provision`.

File: README.rst (hunk 1 above)
Comment (OmarIthawi) [reply README.rst]: The clone part is needed, thanks @johnbaldwin. But the `make dev.checkout` isn't.

File: README.rst (hunk 2 above)
Comment (OmarIthawi) [reply README.rst]: @johnbaldwin @thraxil thanks for the report. I've found and fixed the error which used to break the step `$ make lms-static`.

Please checkout this pull request https://github.com/appsembler/edx-platform/pull/375 and let me know.

**Comment by OmarIthawi** [comment]
@johnbaldwin @thraxil thanks for the review. This branch kinda worked for both @grozdanowski and @melvinsoft so I've merged it.

**Pull request merged**